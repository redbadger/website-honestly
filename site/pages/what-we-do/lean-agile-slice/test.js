import React from 'react';
import { mount } from 'enzyme';

import LeanAgileSlice from './index';

describe('site/pages/our-work/case-study/sky/', () => {
  it('renders correctly', () => {
    expect(mount(<LeanAgileSlice />)).toMatchSnapshot();
  });
});
