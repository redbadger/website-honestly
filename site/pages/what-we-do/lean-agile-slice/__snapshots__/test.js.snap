// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`site/pages/our-work/case-study/sky/ renders correctly 1`] = `
<LeanAgileSlice>
  <section
    className="LeanAgileSlice"
  >
    <h2
      className="leanAgileHeader"
    >
      Our flexible lean and agile process
    </h2>
    <div
      className="steps"
    >
      <div
        className="loopContainer"
      >
        <SmallLoop
          className="smallLoopImage"
        >
          <svg
            className="smallLoopImage"
            viewBox="0 0 298 127"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              fill="none"
              stroke="#B88DFF"
              strokeWidth="5"
            >
              <path
                d="M148.692 63.01l-42.733-42.787c-23.601-23.63-61.865-23.63-85.466 0C8.59 32.142 2.69 47.978 2.793 63.598m145.899-.588l42.732 42.789c23.601 23.63 61.866 23.63 85.466 0 11.957-11.973 17.856-27.685 17.697-43.376"
              />
              <path
                d="M148.692 63.01l-42.733 42.79c-23.601 23.63-61.865 23.63-85.466 0C8.59 93.88 2.69 78.044 2.793 62.423"
                strokeDasharray="17.5,10.5,17.5,17.5"
              />
              <path
                d="M148.692 63.01l42.732-42.787c23.601-23.63 61.866-23.63 85.466 0 11.957 11.973 17.856 27.684 17.697 43.375"
                strokeDasharray="17.5,10.5,17.5,10.5"
              />
            </g>
          </svg>
        </SmallLoop>
        <LargeLoop
          className="largeLoopImage"
        >
          <svg
            className="largeLoopImage"
            viewBox="0 0 422 1004"
            xmlns="http://www.w3.org/2000/svg"
          >
            <g
              fill="none"
              stroke="#B88DFF"
              strokeWidth="10"
            >
              <path
                d="M211 502l145.664-145.57c80.448-80.395 80.448-210.742 0-291.136C316.09 24.744 262.178 4.647 209 4.996M211 502L65.336 647.566c-80.448 80.398-80.448 210.744 0 291.138 40.758 40.731 94.246 60.826 147.664 60.285"
              />
              <path
                d="M211 502L65.336 356.43c-80.448-80.395-80.448-210.742 0-291.136C105.91 24.744 159.822 4.647 213 4.996"
                strokeDasharray="25,15,25,25"
              />
              <path
                d="M211 502l145.664 145.566c80.448 80.398 80.448 210.744 0 291.138-40.758 40.731-94.246 60.826-147.664 60.285"
                strokeDasharray="25,15,25,15"
              />
            </g>
          </svg>
        </LargeLoop>
        <div
          className="wordLeft"
        >
          Innovation
        </div>
        <div
          className="wordRight"
        >
          Delivery
        </div>
      </div>
      <div>
        <h3
          className="stepInnovationHeader"
        >
          Innovation
        </h3>
        <div
          className="step"
          key="Research"
        >
          <div
            className="marker"
          />
          <div
            className="stepContent"
          >
            <h4
              className="stepSubHeader"
            >
              Research
            </h4>
            <p
              className="stepDescription"
            >
              We’ll understand your business and your target audience
            </p>
          </div>
        </div>
        <div
          className="step"
          key="Ideate"
        >
          <div
            className="marker"
          />
          <div
            className="stepContent"
          >
            <h4
              className="stepSubHeader"
            >
              Ideate
            </h4>
            <p
              className="stepDescription"
            >
              Together we’ll define the problem to be solved and develop a range of possible solutions
            </p>
          </div>
        </div>
        <div
          className="step"
          key="Test"
        >
          <div
            className="marker"
          />
          <div
            className="stepContent"
          >
            <h4
              className="stepSubHeader"
            >
              Test
            </h4>
            <p
              className="stepDescription"
            >
              Let’s learn fast and cheaply discard unfeasible solutions
            </p>
          </div>
        </div>
        <h3
          className="stepDeliveryHeader"
        >
          Delivery
        </h3>
        <div
          className="step"
          key="Backlog"
        >
          <div
            className="marker"
          />
          <div
            className="stepContent"
          >
            <h4
              className="stepSubHeader"
            >
              Backlog
            </h4>
            <p
              className="stepDescription"
            >
              Together we’ll prioritise and validate solutions as concrete initiatives
            </p>
          </div>
        </div>
        <div
          className="step"
          key="Execute"
        >
          <div
            className="marker"
          />
          <div
            className="stepContent"
          >
            <h4
              className="stepSubHeader"
            >
              Execute
            </h4>
            <p
              className="stepDescription"
            >
              We’ll deliver great quality products and services with speed
            </p>
          </div>
        </div>
        <div
          className="step"
          key="Optimise"
        >
          <div
            className="marker"
          />
          <div
            className="stepContent"
          >
            <h4
              className="stepSubHeader"
            >
              Optimise
            </h4>
            <p
              className="stepDescription"
            >
              Together we’ll constantly measure and learn to look out for new opportunities and improvements
            </p>
          </div>
        </div>
        <div
          className="stepPaddingBottom"
        />
      </div>
    </div>
  </section>
</LeanAgileSlice>
`;
